extensions/blocks/map/save.js

```diff
@@ -28,6 +37,7 @@ class MapSave extends Component {
 				data-zoom={ zoom }
 				data-map-center={ JSON.stringify( mapCenter ) }
 				data-marker-color={ markerColor }
+				data-scroll-to-zoom={ scrollToZoom ? scrollToZoom : null }
```

Comment:
We could simplify this as `scrollToZoom || null`—maybe `undefined` is more correct here? 🤔

Reply:
Aye, I'll simplify it a little bit. I went with `null` as it's also used for the `alignClassName` asssignment above, and it appears to have the desired effect: if the `scrollToZoom` option isn't set, `data-scroll-to-zoom` won't be added to the output HTML.

Reply:
Tweaked in 1b31139ef.

Reply:
Yup, I did notice the behaviour was identical with `null` and `undefined`, but for some reasons I was sure only `undefined` would omit the prop. 🤷‍♂️